src/applications/personalization/profile-2/components/personal-information/PersonalInformation.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from 'react';
+
+import { focusElement } from 'platform/utilities/ui';
+
+import PersonalInformationContent from './PersonalInformationContent';
+
+const PersonalInformation = () => {
+  useEffect(() => {
+    focusElement('[data-focus-target]');
+  }, []);
```

Comment:
I used `useEffect` like this once while messing around with Create React App. I got a great big linter error about the `[]` passed in. This is a totally legit way to do the same thing as `componentDidMount` though, right? For my own learning 🙂

Reply:
yeah, this is how you do `componentDidMount`, more or less